src/plone/api/content.py

```diff
@@ -549,12 +549,11 @@ def find(context=None, depth=None, **kwargs):
     # Convert interfaces to their identifiers
     object_provides = query.get('object_provides', [])
     if object_provides:
-        if not isinstance(object_provides, list):
+        if not isinstance(object_provides, (list, tuple)):
```

Comment:
Yeah, you are right, this is not the best possible solution.

Anyway checking for collections.Iterable leads you to problem in case of dicts and strings:
```
>>> a = {}
>>> b =''
>>> import collections
>>> isinstance(a, collections.Iterable)
True
>>> isinstance(b, collections.Iterable)
True
```

This would lead to additional "if"s :)